content/docs/pipelines/pipelines-deploy-alternatives.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Pipelines Deploy Alternatives"
+description = "Alternative approaches to deploy lightweight Kubeflow Pipelines to a cluster"
+weight = 15
++++
+
+Despite [deploying Kubeflow](/docs/started/getting-started/#installing-kubeflow) as a
+whole with many components including pipelines, you also have a choice to deploy
+Kubeflow Pipelines' lite deployment. Pipelines has a kustomize manifests for
+this. You can follow the instructions below to deploy it in an existing cluster.
+
+Knowledge about [Kubernetes](https://kubernetes.io/docs/home/), [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) and [kustomize](https://kustomize.io/) will help you understand this
+document better and be able to customize based on your needs.
+
+
+## TL;DR
```

Comment:
The heading "TL;DR" here is confusing in two ways. Firstly, it implies that this section is a summary of what follows - but that doesn't seem right, as the first section deploys to an existing cluster whereas the later section involves creating a cluster. Secondly, not everyone knows what "TL;DR" means.

Suggested heading for better clarity:
"Deploying Kubeflow Pipelines to an existing cluster"